gulp-tasks/watch.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @file
+ * This file contains function for watch gulp task
```

Comment:
Lets not have `This file contains`. Instead just describe what it contains
`Exports Gulp "watch" task`

PS: This is similar to not mentioning `This function does X`
Also please check in all files.